cmd/imagec/imagec.go

```diff
@@ -673,6 +674,13 @@ func main() {
 		log.Debugf("Running standalone")
 	}
 
+	// track whether the image is from a custom registry or not
+	registryURL := fmt.Sprintf("%s/", options.registry)
```

Comment:
Is this really just to add a trailing slash?  I don't see why we need this.  Can you comment?  Also if this is in fact a URL, you may want to check out the [url](https://golang.org/pkg/net/url/) package.